js/index.js

```diff
@@ -0,0 +1,15 @@
+document.getElementById("card-1").onclick = function() {
+    redirectTo("./faq.html");
+};
+
+document.getElementById("card-2").onclick = function() {
+    redirectTo("./discord.html");
+};
+
+document.getElementById("card-3").onclick = function() {
+    redirectTo("./code-of-conduct.html");
+};
```

Comment:
it can be rewritten as 

```
document.getElementById('cards').addEventListener('click', (e) => {
  if(e.target.hasAttribute('data-url')){
    redirectTo(e.target.getAttribute('data-url'));
  }
});
```